packages/web-frontend/src/sagas/handlers/handleUserPage.js

```diff
@@ -12,9 +12,12 @@ import {
   URL_COMPONENTS,
   VERIFY_EMAIL_PREFIX,
 } from '../../historyNavigation/constants';
+import { isMobile } from '../../utils';
 
 export function* handleUserPage(userPage, initialComponents) {
-  yield put(setOverlayComponent(LANDING));
+  // Don't show the overlay on mobile if the user is on the password reset page, because the overlay component appears over the top of the password reset page
+  const shouldDisplayOverlay = !isMobile() || userPage !== PASSWORD_RESET_PREFIX;
+  if (shouldDisplayOverlay) yield put(setOverlayComponent(LANDING));
```

Comment:
This is a good solution. I think the ideal flow would be that the user is taken to the Landing screen after the password reset form is done. Is that easily possible to add in?

Reply:
I have made changes to the flow so that:
- The project modal doesn't appear when reset password and/or change password modals are open
- On closing the reset password/change password modals, if a project is not currently selected, 'go home'
- Does this for both mobile and web, because on web you also would have a situation of potentially not having a project selected if you dismiss the reset password modal AND the project modal